package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+        forcefield : {"CHARMM27", "GLYCAM06", "other"} (optional)
+            Name of the forcefield used. Switches between different
+            :attr:`~HydrogenBondAnalysis.DEFAULT_DONORS` and
+            :attr:`~HydrogenBondAnalysis.DEFAULT_ACCEPTORS` values.
+            ["CHARMM27"]
+        donors : sequence (optional)
+            Extra H donor atom types (in addition to those in
+            :attr:`~HydrogenBondAnalysis.DEFAULT_DONORS`), must be a sequence.
+        acceptors : sequence (optional)
+            Extra H acceptor atom types (in addition to those in
+            :attr:`~HydrogenBondAnalysis.DEFAULT_ACCEPTORS`), must be a sequence.
+        start : int (optional)
+            starting frame-index for analysis, ``None`` is the first one, 0.
+            `start` and `stop` are 0-based frame indices and are used to slice
+            the trajectory (if supported) [``None``]
+        stop : int (optional)
+            last trajectory frame for analysis, ``None`` is the last one [``None``]
+        step : int (optional)
+            read every `step` between `start` (included) and `stop` (excluded),
+            ``None`` selects 1. [``None``]
+        detect_hydrogens : {"distance", "heuristic"} (optional)
+            Determine the algorithm to find hydrogens connected to donor
+            atoms. Can be "distance" (default; finds all hydrogens in the
+            donor's residue within a cutoff of the donor) or "heuristic"
+            (looks for the next few atoms in the atom list). "distance" should
+            always give the correct answer but "heuristic" is faster,
+            especially when the donor list is updated each
+            for each frame. ["distance"]
+        distance_type : {"hydrogen", "heavy"} (optional)
+            Measure hydrogen bond lengths between donor and acceptor heavy
+            attoms ("heavy") or between donor hydrogen and acceptor heavy
+            atom ("hydrogen"). If using "heavy" then one should set the *distance*
+            cutoff to a higher value such as 3.5 Å. ["hydrogen"]
+        debug : bool (optional)
+            If set to ``True`` enables per-frame debug logging. This is disabled
+            by default because it generates a very large amount of output in
+            the log file. (Note that a logger must have been started to see
+            the output, e.g. using :func:`MDAnalysis.start_logging`.)
+        verbose : bool (optional)
+            Toggle progress output. (Can also be given as keyword argument to
+            :meth:`run`.)
+
+        Raises
+        ------
+        :exc:`SelectionError`
+            is raised for each static selection without the required
+            donors and/or acceptors.
+
+        Notes
+        -----
+        In order to speed up processing, atoms are filtered by a coarse
+        distance criterion before a detailed hydrogen bonding analysis is
+        performed (`filter_first` = ``True``). This is now the default.
```

Comment:
remove "This is now the default."

Reply:
@orbeckst Removed. Thank you for pointing out.